codecs/src/aper/encode/encode_internal.rs

```diff
@@ -150,4 +154,12 @@ mod tests {
         encode_normally_small_length_determinent(&mut data, 32).unwrap();
         assert_eq!(data.into_bytes(), [0x3e]);
     }
+
+    #[test]
+    fn encode_int_range_256() {
+        let mut data = AperCodecData::new();
+        data.encode_bool(true);
```

Comment:
I am not sure why this `encode_bool` is required here? Is this just to test that we can indeed  align for a full range? If that's the case it's fine.

Reply:
Yes, it is to bump forward the encoding offset so that the encode int function is forced to move onto the second byte.